app/(normalLayout)/definition/page.tsx

```diff
@@ -167,6 +167,51 @@ export default async function Page() {
                   </ul>
                 </div>
               </div>
+              <div className="fr-col-12 fr-col-md-8">
+                <div className="block block--yellow block--fill">
+                  <h3 className="block__title">
+                    Réalisez un signalement sur un bâtiment
+                  </h3>
+
+                  <p>
+                    <strong>
+                      Vous constatez une erreur (bâtiment absent ou mal placé,
+                      erreur d&apos;adresse, etc) ?
+                    </strong>
+                  </p>
+
+                  <span>
+                    Envoyez votre signalement directement sur la{' '}
+                    <a href="https://rnb.beta.gouv.fr/carte">cartographie</a> du
```

Comment:
"carte" à place de "cartographie" 😅
```suggestion
                    <a href="https://rnb.beta.gouv.fr/carte">carte</a> du
```